.github/workflows/jobs.yml

```diff
@@ -252,6 +252,16 @@ jobs:
           make check
   windows_latest_cmake:
     runs-on: windows-latest
+    name: Windows ${{ matrix.msystem }} INTERFACE64=${{ matrix.int64 }}/MPI=${{ matrix.mpi }}
+    strategy:
+      fail-fast: false
```

Comment:
Are you sure this fails if the job fails? (red CI) If not, please remove this, thanks

Reply:
I don't understand your question well.
GHA aborts all jobs of a matrix if any of them fails. This will let them run even one of them fails.

Reply:
> GHA aborts all jobs of a matrix if any of them fails.

OK, so if 1 job fails, the others will still run, but, the final CI status will be "fail", right?

Reply:
Yes, https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#handling-failures